docs/spec.yml

```diff
@@ -299,11 +319,272 @@ paths:
         '403':
           description: Unauthorised Access
         '404':
-          description: Operation not supported
+          description: No records found matching identifier and criteria 
         '422':
           description: Unprocessable request
         '500':
           description: Validation Error
+  '/v3/technical-records/updateVrm/{systemNumber}/{createdTimestamp}':
+    patch:
+      summary: 'Update VRM on a technical record'
+      tags:
+        - Update VRM
+      parameters:
+        - in: path
+          name: systemNumber
+          schema:
+            type: string
+            minLength: 3
+            maxLength: 21
+          required: true
+          description: This represents the systemNumber of the vehicle. Partition key of the DynamoDB table.
+        - in: path
+          name: createdTimestamp
+          schema:
+            type: string
+            pattern: \d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{0,20}Z
+          required: true
+          description: String representation of a DateTime at which the technical record was created. Sort key of the DynamoDB table.
+      operationId: updateVrm
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object 
+              required:
+                - newVrm
+              properties:
+                newVrm:
+                  type: string
+                isCherishedTransfer:
+                  type: boolean
+                thirdMark:
+                  type: string
+      responses:
+        '200':
+          description: Returns the updated technical record
+          content:
+            application/json:
+              schema:
+                oneof:
```

Comment:
This is the first one I found, but I think they all need to be `oneOf` rather than `oneof`

When the doc is generated with `oneof` and you click on the schema tab it says 'any' rather than listing the schemas.